docs/sources/reference/api/docker_remote_api_v1.17.md

```diff
@@ -124,6 +124,11 @@ Create a container
              "OpenStdin":false,
              "StdinOnce":false,
              "Env":null,
+             "Labels": [
+                     "Vendor=Acme",
+                     "License=GPL",
+                     "Version=1.0"
+             ],
```

Comment:
Lolz. Reading back your comment, you probably suggested to *dont* add a value to the array at all?

Should probably be fine as well. There's no need to know how many times I set a label, so `--label foo` will result in {"foo":[]}` (an empty array), right?